site/plugins.json

```diff
@@ -86,7 +86,19 @@
     "name": "Visual diff (Applitools)",
     "package": "netlify-plugin-visual-diff",
     "repo": "https://github.com/applitools/netlify-plugin-visual-diff",
-    "version": "1.2.0"
+    "version": "1.5.0",
+    "compatibility": [
+      {
+        "version": "1.5.0"
```

Comment:
Small note: shouldn't this be a major release, since dropping Node.js version is a breaking change?

Reply:
yup my mistake, about to submit a fix